index.html

```diff
@@ -154,6 +154,39 @@ <h1>About me</h1>
                 </li>
             </ul>
         </section>
+        <footer>
+            <div class="section contact">
+                <h2>
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h2>
+                <form action="https://formspree.io/f/xlezgrpw" method="post">
+                    <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+                    <input type="email" name="email" placeholder="Email address" required>
+                    <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+                    <button type="submit">Get in touch</button>
+                </form>
```

Comment:
The form is well validated in reference to the project requirements! Good job 👍🏻.